gax-java/gax/pom.xml

```diff
@@ -93,6 +93,14 @@
           </excludes>
         </configuration>
       </plugin>
+      <plugin>
+        <!-- Troubleshooting a flaky test in https://github.com/googleapis/sdk-platform-java/issues/1931 -->
```

Comment:
I keep this just in case the problem occurs and we have the logging. This surefire (unit test) configuration does not affect the released libraries to users.